packages/components/src/custom-select-control-v2/stories/index.story.tsx

```diff
@@ -0,0 +1,149 @@
+/**
+ * External dependencies
+ */
+import type { Meta, StoryFn } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { CustomSelect, CustomSelectItem } from '..';
+
+const meta: Meta< typeof CustomSelect > = {
+	title: 'Components (Experimental)/CustomSelectControl v2',
+	component: CustomSelect,
+	subcomponents: {
+		// @ts-expect-error - See https://github.com/storybookjs/storybook/issues/23170
+		CustomSelectItem,
+	},
+	argTypes: {
+		children: { control: { type: null } },
+	},
+	parameters: {
+		actions: { argTypesRegex: '^on.*' },
+		controls: { expanded: true },
+		docs: {
+			canvas: { sourceState: 'shown' },
+			source: { excludeDecorators: true },
+		},
+	},
+	decorators: [
+		( Story ) => (
+			<div
+				style={ {
+					minHeight: '250px',
+				} }
+			>
+				<Story />
+			</div>
+		),
+	],
+};
+export default meta;
+
+const Template: StoryFn< typeof CustomSelect > = () => {
+	return (
+		<CustomSelect label="Label" defaultValue="Default">
+			<CustomSelectItem value="Small">
+				<span style={ { fontSize: '75%' } }>Small</span>
+			</CustomSelectItem>
+			<CustomSelectItem value="Default">Default</CustomSelectItem>
```

Comment:
Prior to commit https://github.com/WordPress/gutenberg/pull/55790/commits/c3e4d29e22045299b5fb3dc0c872bec6acf301ac, this was: `<CustomSelectItem value="Default"/>` 

The children didn't need to be added because the value could be used instead. However, when adding the checkmark `<Ariakit.SelectItemCheck/>`, I had to explicitly define children rather than relying on `...props`: 

https://github.com/WordPress/gutenberg/blob/f771fadea89d993ec09858e425942e751d89bb3c/packages/components/src/custom-select-control-v2/index.tsx#L78-L84

It may be a good follow-up if we want children to be optional (unless it seems like something that should be in the MVP).

Reply:
Have you considered still rendering the `value` as a fallback for the `children` ?

Something like `{ children ?? props.value }` should do it, allowing us to keep `children` as optional (and go back to `<CustomSelectItem value="Default" />`)